tasks.py

```diff
@@ -426,34 +437,27 @@ def unittest_coverage(context):
     run_command(context, command)
 
 
-@task
-def integration_tests(context):
-    """Some very generic high level integration tests."""
-    session = requests.Session()
-    retries = 1
-    max_retries = 60
-
-    start(context)
-    while retries < max_retries:
-        try:
-            response = session.get("http://localhost:8080", timeout=5)
-        except requests.exceptions.ConnectionError:
-            print("Nautobot not ready yet sleeping for 5 seconds...")
-            sleep(5)
-            retries += 1
-            continue
-        if response.ok:
-            print("Nautobot is ready...")
-            break
-        else:
-            raise Exit(f"Nautobot returned and invalid status {response.status_code}", 1)
-    if retries >= max_retries:
-        raise Exit("Timed out waiting for Nautobot", 1)
+@task(
+    help={
+        "keepdb": "Save and re-use test database between test runs for faster re-testing.",
+        "label": "Specify a directory or module to test instead of running all Nautobot tests.",
+        "failfast": "Fail as soon as a single test fails don't run the entire test suite.",
+        "buffer": "Discard output from passing tests.",
+        "verbose": "Enable verbose test output.",
+    }
+)
+def integration_test(
+    context, keepdb=False, label="nautobot.core.tests.integration", failfast=False, buffer=True, verbose=False
+):
+    """Run Nautobot integration tests."""
+    # Append extra args to the context that will be picked up by the `unittest` task.
+    context.extra_args = " --tag integration"
```

Comment:
Did you consider instead adding a `tag` parameter to the `unittest` task? `invoke integration-test` would then presumably just call `unittest(context, keepdb=keepdb, label=label, failfast=failfast, buffer=buffer, verbose=verbose, tag="integration")`

Reply:
I considered it, but since we don't have any use-cases for tags on unit tests right now I just took the cheap way out.

Reply:
Ok. This approach feels a bit inelegant to me compared to that alternative, but I won't raise a stink about it. :-)

Reply:
Never mind, I see you changed it - thanks! :-)